lighthouse-core/gather/gather-runner.js

```diff
@@ -169,6 +170,12 @@ class GatherRunner {
       options.flags = {};
     }
 
+    if (typeof options.config === 'undefined') {
```

Comment:
Why? The Gatherer needs the config, it should be the one to assert that it exists.